roles/moodle/templates/moodle-nginx.conf.j2

```diff
@@ -1,13 +1,32 @@
-location /moodle {
-    proxy_set_header X-Real-IP  $remote_addr;
-    proxy_set_header X-Forwarded-For $remote_addr;
-    proxy_set_header Host $host;
-    proxy_pass http://127.0.0.1:{{ apache_port }};
+# 2021-07-02: Let's monitor & learn from these 2 pages year-by-year:
+# https://docs.moodle.org/311/en/Nginx
+# https://github.com/moodlebox/moodlebox/blob/master/roles/webserver/templates/etc/nginx/sites-available/default.j2
+
+# Odd that MoodleBox.net is using only the 2nd half of https://docs.moodle.org/311/en/Nginx#XSendfile_aka_X-Accel-Redirect ?
```

Comment:
If I understand correctly the comment: First half is done here https://github.com/moodlebox/moodlebox/blob/851d1eb2fd3672e275ca2512eabf7eeed79efe2b/roles/moodleinstall/tasks/coreinstall.yml#L102-L103

Reply:
Could use this link for our install.yml?

Reply:
Yes (as far as i'm concerned)